model_compression_toolkit/pytorch/back2framework/model_builder.py

```diff
@@ -183,7 +183,8 @@ def forward(self,
 def model_builder(graph: common.Graph,
                   mode: ModelBuilderMode = ModelBuilderMode.QUANTIZED,
                   append2output: List[Any] = None,
-                  fw_info: FrameworkInfo = DEFAULT_PYTORCH_INFO) -> Tuple[torch.nn.Module, Any]:
+                  fw_info: FrameworkInfo = DEFAULT_PYTORCH_INFO,
+                  gptq_config: GradientPTQConfig = None) -> Tuple[torch.nn.Module, Any]:
     """
```

Comment:
Why does the model builder need to receive a gptq config?

Reply:
Because when we build the model for GPTQ we need to define QuantizeWrapper or some other wrapper which hold GPTQ configuration